### ISSUE TITLE
FirebaseNameOrConfigToken & FirebaseOptionsToken weren't found on @angular/fire

### ISSUE THREAD

**Issue by coder80s** [AnthonyNahas/ngx-auth-firebaseui#514]
### Bug Report or Feature Request (mark with an `x`)
```
- [X] bug report
- [ ] feature request
```

### OS and Version?
> Windows 10

### Versions
@angular-devkit/architect         0.901.1
@angular-devkit/build-angular     0.901.1
@angular-devkit/build-optimizer   0.901.1
@angular-devkit/build-webpack     0.901.1
@angular-devkit/core              9.1.1
@angular-devkit/schematics        9.1.1
@angular/cdk                      9.2.0
@angular/fire                     6.0.0
@angular/flex-layout              9.0.0-beta.29
@angular/material                 9.2.0
@ngtools/webpack                  9.1.1
@schematics/angular               9.1.1
@schematics/update                0.901.1
rxjs                              6.5.5
typescript                        3.8.3
webpack                           4.42.0

### Repro steps
My project was initially based on ngrx-rocket: https://github.com/ngx-rocket/generator-ngx-rocket which was working fine and then I follow install instructions of this plugin, step by step using npm approach.

Then the following error is given when running project.

### The log given by the failure
ERROR in ./node_modules/ngx-auth-firebaseui/__ivy_ngcc__/fesm2015/ngx-auth-firebaseui.js 4179:29-54
"export 'FirebaseNameOrConfigToken' was not found in '@angular/fire'
ERROR in ./node_modules/ngx-auth-firebaseui/__ivy_ngcc__/fesm2015/ngx-auth-firebaseui.js 4175:29-49
"export 'FirebaseOptionsToken' was not found in '@angular/fire'

### Desired functionality
I want to be able to use this plugin as it would ease register/signin/signout, etc processes related to FireBase and I wanted to introduce it as one of the steps to register potential users.

### Mention any other details that might be useful
I've invited @AnthonyNahas to be able to access to my private GitHub Repo

**Comment by zkyy** [comment]
Yes, I've also had the problem yesterday. Complete new Angular 9.1 project and couldn't build the project.

**Comment by lazyfahim** [comment]
I have already found the problem my Package.Json is 



`{
  "name": "one-start",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular-material-extensions/password-strength": "^6.0.0",
    "@angular/animations": "^9.1.1",
    "@angular/cdk": "^9.2.1",
    "@angular/common": "~9.1.1",
    "@angular/compiler": "~9.1.1",
    "@angular/core": "~9.1.1",
    "@angular/fire": "^6.0.0",
    "@angular/flex-layout": "^9.0.0-beta.29",
    "@angular/forms": "^9.1.1",
    "@angular/material": "^9.2.1",
    "@angular/platform-browser": "~9.1.1",
    "@angular/platform-browser-dynamic": "~9.1.1",
    "@angular/router": "^9.1.1",
    "firebase": "^7.14.0",
    "firebaseui": "^4.5.0",
    "firebaseui-angular": "^4.1.0",
    "ngx-auth-firebaseui": "^4.0.4",
    "rxjs": "~6.5.4",
    "tslib": "^1.10.0",
    "zone.js": "~0.10.2"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.901.1",
    "@angular/cli": "~9.1.1",
    "@angular/compiler-cli": "~9.1.1",
    "@angular/language-service": "~9.1.1",
    "@types/node": "^12.11.1",
    "@types/jasmine": "~3.5.0",
    "@types/jasminewd2": "~2.0.3",
    "codelyzer": "^5.1.2",
    "jasmine-core": "~3.5.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~4.4.1",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage-istanbul-reporter": "~2.1.0",
    "karma-jasmine": "~3.0.1",
    "karma-jasmine-html-reporter": "^1.4.2",
    "protractor": "~5.4.3",
    "ts-node": "~8.3.0",
    "tslint": "~6.1.0",
    "typescript": "~3.8.3"
  }
}
`

**Comment by mmorillo127** [comment]
Same error, almost:

> ERROR in ./node_modules/ngx-auth-firebaseui/__ivy_ngcc__/fesm2015/ngx-auth-firebaseui.js **3694**:29-54
> "export 'FirebaseNameOrConfigToken' was not found in '@angular/fire'
> ERROR in ./node_modules/ngx-auth-firebaseui/__ivy_ngcc__/fesm2015/ngx-auth-firebaseui.js **3690**:29-49
> "export 'FirebaseOptionsToken' was not found in '@angular/fire'

Part of my package.json
```
dependencies": {
    "@angular-material-extensions/password-strength": "^6.0.0",
    "@angular/animations": "~9.1.1",
    "@angular/cdk": "^9.2.1",
    "@angular/common": "~9.1.1",
    "@angular/compiler": "~9.1.1",
    "@angular/core": "~9.1.1",
    "@angular/fire": "^6.0.0-rc.2",
    "@angular/flex-layout": "^9.0.0-beta.29",
    "@angular/forms": "~9.1.1",
    "@angular/material": "^9.2.1",
    "@angular/platform-browser": "~9.1.1",
    "@angular/platform-browser-dynamic": "~9.1.1",
    "@angular/router": "~9.1.1",
    "firebase": "^7.13.2",
    "ngx-auth-firebaseui": "^4.0.4",
    "rxjs": "~6.5.4",
    "tslib": "^1.10.0",
    "zone.js": "~0.10.2"
  }
```

**Comment by kevin192291** [comment]
I am getting the same error:
```
"dependencies": {
    "@angular-material-extensions/password-strength": "^6.0.0",
    "@angular/animations": "~9.0.5",
    "@angular/cdk": "^9.2.0",
    "@angular/common": "~9.0.5",
    "@angular/compiler": "~9.0.5",
    "@angular/core": "~9.0.5",
    "@angular/fire": "^6.0.0",
    "@angular/flex-layout": "^9.0.0-beta.29",
    "@angular/forms": "~9.0.5",
    "@angular/material": "^9.2.0",
    "@angular/platform-browser": "~9.0.5",
    "@angular/platform-browser-dynamic": "~9.0.5",
    "@angular/router": "~9.0.5",
    "@asymmetrik/ngx-leaflet": "^6.0.1",
    "@asymmetrik/ngx-leaflet-draw": "^5.0.1",
    "firebase": "^7.14.0",
    "leaflet": "^1.6.0",
    "leaflet-draw": "^1.0.4",
    "material-design-icons": "^3.0.1",
    "ngx-auth-firebaseui": "^4.0.4",
    "rxjs": "~6.5.4",
    "tslib": "^1.10.0",
    "zone.js": "~0.10.2"
  },
```

**Comment by jnorkus** [comment]
https://github.com/AnthonyNahas/ngx-auth-firebaseui/issues/466

**Comment by AnthonyNahas** [comment]
this has been fixed with v4.1.0
